cog/perk.py

```
@@ -0,0 +1,321 @@
+                # 沒指定色碼就只改名，保留原本的顏色
+                await role.edit(name=name)
+            else:
+                await role.edit(name=name, colour=colour)
+            await ctx.author.add_roles(role)
+        except discord.Forbidden:
+            await ctx.respond("我沒有管理身分組的權限，請聯絡管理員！")
+            end(connection, cursor)
+            return
+
+        # 續租從原到期日往後算，新租從現在起算
+        now = datetime.now()
```

Ruff (0.15.20)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)